src/app/components/header/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Link = styled.a`
+  font-family: ReithSans, Arial, Helvetica, freesans, sans-serif;
+  font-size: 2.2em;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+`;
+
+const StyledHeader = styled.header`
+  background-color: #bb1919;
+  height: 40px;
+  width: 100%;
+  padding: 15px 16px;
```

Comment:
We'd like to keep padding and margins to be multiples of 8px. So if you could update this to be `padding: 16px` that would be great.

Reply:
Done